app/src/main/java/io/github/droidkaigi/confsched/util/DataBindingAttributeUtil.java

```diff
@@ -22,15 +22,16 @@
 
 public class DataBindingAttributeUtil {
 
-    @BindingAdapter("speakerImageUrl")
-    public static void setSpeakerImageUrl(ImageView imageView, @Nullable String imageUrl) {
+    @BindingAdapter({"speakerImageUrl", "speakerImageSize"})
+    public static void setSpeakerImageUrlWithSize(ImageView imageView, @Nullable String imageUrl, float sizeInDimen) {
         if (TextUtils.isEmpty(imageUrl)) {
             imageView.setImageDrawable(ContextCompat.getDrawable(imageView.getContext(), R.drawable.ic_speaker_placeholder));
         } else {
+            final int size = (int)(sizeInDimen + 0.5f);
```

Comment:
No, it is just for rounding up the float value to int.  I read the similar code in `TypedValue#complextToDimensionPixelSize()` as below.

```
    {
        final float value = complexToFloat(data);
        final float f = applyDimension(
                (data>>COMPLEX_UNIT_SHIFT)&COMPLEX_UNIT_MASK,
                value,
                metrics);
        final int res = (int)(f+0.5f);
        if (res != 0) return res;
        if (value == 0) return 0;
        if (value > 0) return 1;
        return -1;
    }
```

But maybe it is confusing and we should use resource ID, instead.